CHANGES.rst

```diff
@@ -8,6 +8,8 @@ Version 0.0.2   (unreleased)
 
 • Additional optimization algorithms: Linearized ADMM and PDHG.
 • Move optimization algorithms into ``optimize`` subpackage.
+• Additional iteration stats columns for iterative ADMM subproblem solvers.
+• Renamed "Primal Residual" to "Prim Residual" in iteration stats.
```

Comment:
Is that so? Looks like it is `Prim_Rsdl`.

I would have gone with `prml_rsdl` and `dual_rsdl` as in removing vowels from 'primal' and 'residual'.

Also, why the capitalization?

Your call, though.

Reply:
> Is that so? Looks like it is `Prim_Rsdl`.

It's `Prim Rsdl` in the display column header, and `Prim_Rsdl` when accessing the data from the returned iteration statistics.

> I would have gone with `prml_rsdl` and `dual_rsdl` as in removing vowels from 'primal' and 'residual'.

Agreed. I will change it.
 
> Also, why the capitalization?

I think capitalization looks better in the column headers. I guess the default corresponding name in the data record could be converted to lower case, but I'm on the fence as to whether it's worth the effort. How strongly do you feel about this?

Reply:
> It's Prim Rsdl in the display column header, and Prim_Rsdl when accessing the data from the returned iteration statistics.

So I think the CHANGES.rst needs to be updated then.

> I think capitalization looks better in the column headers. I guess the default corresponding name in the data record could be converted to lower case, but I'm on the fence as to whether it's worth the effort. How strongly do you feel about this?

How consistent are we about upper/lower case conventions? If we generally follow the rules we should try to do it here, too.